src/styles/blocks/card.scss

```
@@ -0,0 +1,116 @@
+@import "../../layout_stars/src/style.css";
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.card {
+  display: flex;
+  padding: 0 16px;
+  width: 200px;
+  flex-direction: column;
+  justify-content: center;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;

```

don't forget to create variables for all colors

```suggestion
  border: 1px solid $c-color;

```